packages/reporter/src/main.tsx

```diff
@@ -154,8 +154,10 @@ declare global {
 if (window.Cypress) {
   window.state = appState
   window.render = (props) => {
-    // @ts-ignore
-    render(<Reporter {...props as Required<BaseReporterProps>} />, document.getElementById('app'))
+    const container: HTMLElement = document.getElementById('app') as HTMLElement
```

Comment:
see see https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis. This render is used for cy-in-cy